tests/main/interfaces-daemon-notify/task.yaml

```diff
@@ -19,15 +19,15 @@ execute: |
     snap connect test-snapd-daemon-notify:daemon-notify
 
     echo "Then after we restart the servive there is not denials"
-    denials_before="$(snap logs -n=100 test-snapd-daemon-notify.notify | grep -c 'Permission denied' || true)"
+    denials_before="$(snap logs -n=all test-snapd-daemon-notify.notify | grep -c 'Permission denied' || true)"
     snap stop test-snapd-daemon-notify.notify
```

Comment:
Now looking at the test, this line should be moved just before we do the counting.